### ISSUE TITLE
By 3/6: Building Cohesive Model and Presentation

### ISSUE THREAD

**Issue by abrownjosh** [rvelez1005/SARG-Defender-Analytics#5]
- [ ] Using a grid search finding optimal/best hyperparameters
- [ ] Test linear regression, ridge regression, random forest, and gradient boosting + other models if it would help
- [ ] Examine outliers in dataset plot
- [ ] Fix model bias from seasonal data -> season-specific rating
- [ ] Fix leakage from hyperparameter tuning on both training and testing (speak with Tad)
- [ ] Finish draft of Materials/Methods for the paper.
- [ ] Set up tableau to the point where we know what we need to do with it
